services/web/server/src/simcore_service_webserver/statics/_events.py

```diff
@@ -79,10 +79,6 @@ async def create_cached_indexes(app: web.Application) -> None:
             msg = f"Could not fetch index at {url!s}. Stopping application boot"
             raise RuntimeError(msg) from err
 
-        # fixes relative paths
```

Comment:
nice!

Reply:
Please do not forget to create an issue/PR to follow up on redirecting the root path `/` to the static-webserver instead of the webserver.  It would require a small change on the web-server but also configurations in the reverse proxy.

Reply:
I just undid the change.